lib/data_update_scripts/20201001173841_add_navigation_links.rb

```diff
@@ -0,0 +1,26 @@
+module DataUpdateScripts
+  class AddNavigationLinks
+    PROTOCOL = ApplicationConfig["APP_PROTOCOL"].freeze
+    DOMAIN = Rails.application&.initialized? ? SiteConfig.app_domain : ApplicationConfig["APP_DOMAIN"]
+    BASE_URL = "#{PROTOCOL}#{DOMAIN}".freeze
+
+    READING_ICON = File.read(Rails.root.join("app/assets/images/twemoji/drawer.svg")).freeze
```

Comment:
Thinking outloud here..... I think it's great that we're giving individual forems a lot of control over this section. There's an argument for whether we want to be more restrictive for consistency _across_ forems, but I think this is the right baseline here: Focus on the ways Forems can independently succeed through flexibility in places like this, but in the future I could see a scenario where users of forems could customize this space such that they always see reading list in the same place, or whatever.... This would be in what we now have as `/settings/ux` And that they could port that customization across all their forems to automatically inject their preferences.

Not a blocker, just a thought.

Reply:
I feel like even the possibility of a scenario as portable preferences should be documented somewhere (even if it's as an aspirational item in some sort of backlog) and not get lost in a Github comment. The clarity/extensibility of code often relies on the soundness of the data model and shoehorning new features into existing structures can be very painful. I know you're not the biggest fan of big up front designs, and I'm not advocating for that, but I do have concerns over the long-term quality of the codebase if things stay too ad-hoc.